packages/component-library/src/components/plmg-tooltip/plmg-tooltip.stories.js

```diff
@@ -0,0 +1,202 @@
+  let wrapper = document.createElement('div');
+  const htmlContent = `
+  <div style="display: flex; flex-direction: column; height: 100vh;">
+      
+      <p id="element-one" tabindex="1" style="padding: 10px; margin-left: 150px; width: fit-content;">below start</p>
+      <plmg-tooltip target-element="element-one" arrow-side="top" arrow-position="start" content="Bottom with arrow start"></plmg-tooltip>
+      
+      <p id="element-two" tabindex="2" style="padding: 10px; margin-left: 150px; width: fit-content;">below middle</p>
+      <plmg-tooltip target-element="element-two" arrow-side="top" arrow-position="middle" content="Bottom with arrow middle"></plmg-tooltip>
+      
+      <p id="element-three" tabindex="3" style="padding: 10px; margin-left: 150px; width: fit-content;">below end</p>
+      <plmg-tooltip target-element="element-three" arrow-side="top" arrow-position="end" content="Bottom with arrow end"></plmg-tooltip>
+    </div>
+    `;
+  wrapper.innerHTML = htmlContent;
+  return wrapper;
+};
+
+export const TooltipAbove = (args) => {
+  let wrapper = document.createElement('div');
+  const htmlContent = `
+  <div style="display: flex; flex-direction: column; height: 100vh; width: 200px; margin-left: 150px; margin-top: 75px; gap: 20px; width: fit-content;">
+     
+      <p id="element-one" tabindex="1">above start</p>
+      <plmg-tooltip target-element="element-one" arrow-side="bottom" arrow-position="start" content="Tooltip above with arrow start"></plmg-tooltip>
+     
+      <p id="element-two" tabindex="2">above middle</p>
+      <plmg-tooltip target-element="element-two" arrow-side="bottom" arrow-position="middle" content="Tooltip above with arrow middle"></plmg-tooltip>
+     
+      <p id="element-three" tabindex="3">above end</p>
+      <plmg-tooltip target-element="element-three" arrow-side="bottom" arrow-position="end" content="Tooltip above with arrow at the end"></plmg-tooltip>
+    
+    </div>
+    `;
+  wrapper.innerHTML = htmlContent;
+  return wrapper;
+};
+
+export const AllVariations = (args) => {
+  let htmlContent = '';
+  backgroundColors.forEach((backgroundColor) => {
+    arrowSides.forEach((arrowSide) => {
+      arrowPositions.forEach((arrowPosition) => {
+        htmlContent += `
+        <plmg-tooltip force-visible="true" content="${backgroundColor} ${arrowPosition} ${arrowSide}" background-color="${backgroundColor}" arrow-side="${arrowSide}" arrow-position="${arrowPosition}">
+        </plmg-tooltip>
+        `;
+      });
+    });
+  });
+
+  const el = document.createElement('div');
+  el.innerHTML = htmlContent.trim();
+  console.log(el);
+  el.style.display = 'grid';
+  el.style.gridTemplateColumns = 'repeat(3, 1fr)';
+  el.style.gridTemplateRows = 'repeat(5, 1fr)';
+  el.style.gap = '50px';
+  el.style.justifyContent = 'space-between';
+  el.style['flex-wrap'] = 'wrap';
+  return el;
+};
+
+AllVariations.storyName = 'All variations';
+
+// export const AllVariations = (args) => {
```

Comment:
This commented code appears to be a duplicate of the story above. Please remove.